packages/story-editor/src/components/floatingMenu/elements/fontSize.js

```diff
@@ -36,18 +39,31 @@ function FontSize() {
     })
   );
 
-  const handleChange = (value) =>
-    updateSelectedElements({
-      properties: {
-        fontSize: value,
-      },
-    });
+  const pushUpdate = useCallback(
+    (update) => {
+      updateSelectedElements({
+        properties: (element) => {
+          const updates = updateProperties(element, update, true);
+          const sizeUpdates = getUpdatedSizeAndPosition({
+            ...element,
+            ...updates,
+          });
```

Comment:
oh nice, I didn't know this was a thing that existed 👍

Reply:
Me either, Miina pointed me to it.